worker/src/unstract/worker/worker.py

```diff
@@ -112,40 +116,41 @@ def stream_logs(
     ) -> None:
         for line in container.logs(stream=True, follow=True):
             log_message = line.decode().strip()
-            logger.info(f"Log message: {log_message}")
+            self.logger.info(f"[{self.container_name}] - {log_message}")
             self.process_log_message(
                 log_message=log_message,
                 tool_instance_id=tool_instance_id,
                 channel=channel,
             )
 
-    def get_valid_log_message(
-        self, log_message: str, channel: Optional[str] = None
-    ) -> Optional[dict[str, Any]]:
+    def get_valid_log_message(self, log_message: str) -> Optional[dict[str, Any]]:
+        """Get a valid log message from the log message.
+
+        Args:
+            log_message (str): str
+
+        Returns:
+            Optional[dict[str, Any]]: json message
+        """
         try:
             log_dict = json.loads(log_message)
-            if not isinstance(log_dict, dict):
-                logger.error(f"Received invalid log: {log_dict}")
-            else:
+            if isinstance(log_dict, dict):
                 return log_dict
-        except json.JSONDecodeError as e:
-            logger.warn(
-                f"Received invalid JSON log message: {log_message} \n " f"error: {e}"
-            )
-        return None
+        except json.JSONDecodeError:
+            return None
```

Comment:
why is it returning none?

Reply:
The purpose of this function is  to get the execution logs to push it to pubsub for FE logs. If we getting  JSONDecodeError we don't raise the error or logging it, instead we return None. Also we logging the all mesages at stream_logs methods.